compiler/wasm/scripts/install_wasm-pack.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -eu
+# Install wasm-pack 
+cargo install wasm-pack@0.12.1
```

Comment:
This is going to install `wasm-pack` from source rather than downloading a binary. We should prefer `cargo-binstall` if available.

Reply:
Added a check to use (or install) cargo-binstall